src/DataTrail.tsx

```diff
@@ -268,7 +270,7 @@ export class DataTrail extends SceneObjectBase<DataTrailState> implements SceneO
 
         Object.assign(stateUpdate, this.getSceneUpdatesForNewMetricValue(values.metric, nativeHistogramMetric));
       }
-    } else if (values.metric == null) {
+    } else if (values.metric == null && !this.state.embedded) {
```

Comment:
This ensures that when `DataTrail` is initialized with a `metric` in `embedded` mode, the top scene isn't set to `MetricsReducer` when the URL sync happens.